features/forward-declaration.feature

```diff
@@ -14,6 +16,11 @@ Feature: Outputting Value Objects With Forward Declarations
         NSInteger likeCount
         NSUInteger numberOfRatings
         RMProxy* proxy
+        NSArray<RMSomeType *>* followers
+        %import library=HelloKit file=HelloProtocol
```

Comment:
You shouldn't be using import declarations, they are deprecated in favour of the type declarations

Reply:
Unfortunately this looks like how people have been using protocols in existing codebases, because Remodel will try to import "id<HelloProtocol>.h" if you don't specify a %import.

Also, in %type, there is no way to specify it's a protocol so it will be forward declared as @class if UseForwardDeclaration is on.

I will look into fixing these 2 issues as a follow up.